scss/_patterns_side-navigation.scss

```diff
@@ -518,17 +530,22 @@
     }
 
     &:hover,
-    &.is-active,
     &[aria-current='page'],
     &[aria-current='true'] {
-      background: $color-sidenav-item-background-highlight;
+      background: $color-sidenav-item-background-hover;
       color: $color-sidenav-text-active;
     }
 
+    &:active {
+      background: $color-sidenav-item-background-active;
+    }
+
     &.is-active,
     &[aria-current='page'],
     &[aria-current='true'] {
       @include vf-highlight-bar($color-sidenav-item-border-highlight, left);
+
+      background: $color-sidenav-item-background-hover;
```

Comment:
This doesn't seem needed, it's already added in line 535.

Reply:
addressed